alignment/alignment.h

```diff
@@ -129,8 +129,8 @@ int get_matrix(int *matptr, int *xref, int scale) {
 
 void forward_pass(char *ia, char *ib, int n, int m, int *se1, int *se2, int *maxscore, int g, int gh) {
 	int i, j, f, p, t, hh;
-	int HH[MAX_ALN_LENGTH];
-	int DD[MAX_ALN_LENGTH];
+	int * HH = new int[MAX_ALN_LENGTH];
```

Comment:
I'd prefer if you used `std::unique_ptr` instead of explicitly deleting the array below.